src/Provider/FreeGeoIp/FreeGeoIp.php

```diff
@@ -92,4 +100,20 @@ public function getName(): string
     {
         return 'free_geo_ip';
     }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @throws \InvalidArgumentException
+     */
+    protected function getRequest(string $url): RequestInterface
+    {
+        $request = parent::getRequest($url);
+
+        if ($this->locale !== null) {
```

Comment:
Check if `empty($this->locale)` instead. 

https://github.com/geocoder-php/Geocoder/pull/744#discussion_r127926709